src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -147,4 +148,18 @@ public boolean equals(Object obj) {
                 && filteredClientsList.equals(other.filteredClientsList);
     }
 
+    /**
+     * Calculate the total transactions made with clients in the given filtered Client list.
+     *
+     * @param filteredClientList list of Clients
```

Comment:
Great javadocs! Maybe making it consistent for clients whether it should be capitalised would be great :) See lines 152 and 154.